infra/gcb/jenkins_config/coverage_job.xml

```diff
@@ -47,7 +47,11 @@ cd $WORKSPACE/oss-fuzz/infra/gcb
 pip install -r requirements.txt
 project_dir=$WORKSPACE/oss-fuzz/projects/$(basename $JOB_NAME)
 build_id=$(python build_and_run_coverage.py $project_dir)
-python wait_for_build.py $build_id
+if [[ "$build_id" == "NO_FUZZ_TARGETS_FOUND" ]]; then
```

Comment:
An alternative approach could be not to pass empty `$build_id` to `wait_for_build.py`, but I'd rather fail hard if we have an empty `$build_id`.